{{ .ProjectSnake }}/.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
+- repo: local
+  hooks:
+  - id: aspect-format
+    name: aspect-format
+    entry: aspect run format
+    language: system
+    pass_filenames: false
```

Comment:
this seems very wrong - we definitely want pre-commit to tell the formatter which files are modified, otherwise in a big repo it will be too slow in a pre-commit hook.

Reply:
It wouldn't be worth adding back `pass_filenames: false` in light of the above?

Reply:
#76